src/components/app.css

```diff
@@ -0,0 +1,15 @@
+body {
+
```

Comment:
Doesn't really matter right now but we should consider adding reset classes. we use nanoreset in core-insights.

Also, I think we should add a font-size of 8px to the body. This way we can use rem and em to set sizes in our components and easily comply to the 8px grid of sprout. 

Just some thoughts, don't bother with that now. :)

Reply:
I do agree! :)